eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver/DefaultScopeConfigResolver.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\ConfigResolver;
+
+class DefaultScopeConfigResolver extends ContainerConfigResolver
```

Comment:
If I understand the usage intention right, it's `@internal`? Chain should be used in all cases, right?